packages/containers/yarn.lock

```diff
@@ -5165,6 +5152,10 @@ pify@^2.0.0, pify@^2.3.0:
   version "2.3.0"
   resolved "https://registry.yarnpkg.com/pify/-/pify-2.3.0.tgz#ed141a6ac043a849ea588498e7dca8b15330e90c"
 
+pify@^3.0.0:
```

Comment:
:disapointed:
├─┬ sass-loader@6.0.6
│ └── pify@3.0.0
├─┬ jest-cli@20.0.3
│ └── pify@2.3.0

So we got it twice :/ but only for build tools.

Reply:
If we want to update jest we have to update babel and this should be into another PR